src/calibration.c

```diff
@@ -129,13 +130,13 @@ void run_movement_sequence(const char *sequence)
 			move_front();
 			break;
 		case 'L':
-			move_side(LEFT);
+			move_side(LEFT, speed);
```

Comment:
Why don't use move function for left, right, front, back and stop middle?

Reply:
I think we should discuss all this API-related changes in a separate issue.

Reply:
new issue: #352